.github/workflows/github.yml

```diff
@@ -63,6 +72,14 @@ jobs:
         working-directory: plugins/maven/example-client
         run: mvn package
 
+      - name: Build sbt example-client
+        working-directory: plugins/sbt/example-client
+        run:  sbt graphqlCodegen update
+
+      - name: Build sbt-test
+        working-directory: plugins/sbt/sbt-test
+        run: bash test.sh
```

Comment:
normally you would run `sbt scripted` from the `plugins/sbt` directory

Reply:
but gradle or maven plugins also use like `plugins/gradle/example-client` ,`plugins/maven/graphql-java-codegen-maven-plugin`, in this commit, sbt-test is a test project

Reply:
You may be able to check in a symlink at `plugins/sbt/example-client` that takes you to an example client located at `plugins/sbt/src/sbt-test/sbt-graphql-java-codegen/example-client` so that you can both test it in the standard way and follow the directory structure of the other plugins

Reply:
Thank you very much.This test method looks good.